bin/build.js

```diff
@@ -10,12 +10,17 @@ new Generator({
 }).generate()
 
 const entryPoints = ['src/index.tsx']
+
+/**
+ * @type {import('esbuild').BuildOptions}
+ */
 const sharedConfig = {
   entryPoints,
   bundle: true,
   minify: false,
   sourcemap: true,
   tsconfig: './tsconfig.json',
```

Comment:
This documentation [here](https://esbuild.github.io/content-types/#tsconfig-json) is a bit outdated. Still, I believe there are other settings in `tsconfig.json` that are relevant to a build.